assets/xml/objects/object_mask_kerfay.xml

```diff
@@ -1,10 +1,12 @@
 ﻿<Root>
+    <!-- Assets for Kafei's Mask -->
     <File Name="object_mask_kerfay" Segment="10">
-        <DList Name="object_mask_kerfay_DL_000D40" Offset="0xD40" />
-        <Texture Name="object_mask_kerfay_TLUT_001140" OutName="tlut_001140" Format="rgba16" Width="16" Height="16" Offset="0x1140" />
-        <Texture Name="object_mask_kerfay_Tex_001340" OutName="tex_001340" Format="ci8" Width="16" Height="16" Offset="0x1340" />
-        <Texture Name="object_mask_kerfay_Tex_001440" OutName="tex_001440" Format="ci8" Width="8" Height="16" Offset="0x1440" />
-        <Texture Name="object_mask_kerfay_Tex_0014C0" OutName="tex_0014C0" Format="ci8" Width="8" Height="16" Offset="0x14C0" />
-        <Texture Name="object_mask_kerfay_Tex_001540" OutName="tex_001540" Format="ci8" Width="16" Height="8" Offset="0x1540" />
+        <DList Name="gKafeisMaskDL" Offset="0xD40" />
+
+        <Texture Name="gKafeisMaskTLUT" OutName="kafeis_mask_tlut" Format="rgba16" Width="16" Height="16" Offset="0x1140" />
+        <Texture Name="gKafeisMaskHoleTex" OutName="hole" Format="ci8" Width="16" Height="16" Offset="0x1340" TlutOffset="0x1140" />
+        <Texture Name="gKafeisHair1Tex" OutName="hair_1" Format="ci8" Width="8" Height="16" Offset="0x1440" TlutOffset="0x1140" />
+        <Texture Name="gKafeisHair2Tex" OutName="hair_2" Format="ci8" Width="8" Height="16" Offset="0x14C0" TlutOffset="0x1140" />
```

Comment:
I discussed the purpose of these various hair textures on the kafei object PR: https://github.com/zeldaret/mm/pull/1431#discussion_r1359954390
I'm not sure if you want to incorporate that into the mask object, since I'm not sure if they're used the same. But the *textures* are the same.

Reply:
With a quick texture edit seems like they are used the same, so I'll name them the same